docs/direct-debits/Timings.md

```diff
@@ -0,0 +1,25 @@
+---
+stoplight-id: vshzy22o7mgnh
+tags: [Direct Debits]
+---
+
+# Timings
+
+There are several timings to be aware of when setting and taking payments via Direct Debits.
+
+The first is setting up a mandate.  This typically takes 2 business days.  It is performed once for a customer’s bank account.  A new mandate must be set up for their new bank account if a customer changes their bank account.
+
+There are several timings related to taking a payment using Direct Debits.  Direct Debits use the UK’s BACS payment system and thus are subject to the timeframes of the BACS payment system.
+Creating the payment for a customer. 
```

Comment:
Subtitles must have `##` before them, so this would be:
```## Creating the payment for a customer```
The same would apply for other subtitles below